tests/test_salesforce_all_fields_custom_rest.py

```diff
@@ -0,0 +1,48 @@
+"""
+Test that with only custom fields selected for a stream automatic fields and custom fields  are still replicated
+"""
+import copy
+from tap_tester.base_suite_tests.all_fields_test import AllFieldsTest
+from sfbase import SFBaseTest
+from tap_tester.logger import LOGGER
+
+
+class SFCustomFieldsTest(AllFieldsTest, SFBaseTest):
```

Comment:
Is this allowed to be the same class name as the another test?

`test_salesforce_all_fields_custom.py` defines the same class